### ISSUE TITLE
more integration with sxiv

### ISSUE THREAD

**Issue by hektr** [ranger/ranger#1993]
I use sxiv for image viewing. Is it possible to make files marked inside sxiv (using `m` or right mouse-click) tagged in ranger??? I mean, can ranger "watch" what is marked in sxiv and make these files tagged(using default tag)?? Thanks!

**Comment by toonn** [comment]
No. You could have sxiv send its selection to ranger to have it selected via the API maybe (which kinda depends on a WIP PR).

**Comment by chu4ng** [comment]
It is possible with #2151 PR.
